setup.cfg

```diff
@@ -79,6 +79,8 @@ filterwarnings =
     ignore:numpy\.ndarray size changed:RuntimeWarning
     # numpy 1.24 deprecation from skimage (can remove for skimage >= 0.20)
     ignore:`np.bool8` is a deprecated alias for `np.bool_`:DeprecationWarning
+    # TEMP: remove when matplotlib nightly fixes the row_stack deprecations
+    ignore:`row_stack` alias is deprecated:DeprecationWarning
```

Comment:
Hmm why didn't astropy stumble on this? 🤔

Reply:
It's actually fixed in mpl main, but the `devdeps` test picks up the rc version, which does *not* have the fix.

Reply:
Ah, right!

But then again, numpy is also in RC, so it would cancel out for a while. 🤪